history/titles/c_thal_grove.txt

```diff
@@ -1,5 +1,4 @@
 1.1.1={
```

Comment:
Will perish from history after reloading the save if dies

Reply:
Fixed!